apply_recommendations/cmd/apply/main.go

```diff
@@ -13,8 +14,18 @@ import (
 )
 
 func main() {
-	workingDir := flag.String("working-dir", "./", "path to working directory")
-	dryRun := flag.Bool("dry-run", false, "dry run; run terraform plan instead of terraform apply")
+
+	defaultDryRun := false
+	if dryRunEnvVar := os.Getenv("INPUT_DRY-RUN"); dryRunEnvVar != "" {
+		var err error
+		defaultDryRun, err = strconv.ParseBool(dryRunEnvVar)
+		if err != nil {
+			log.Fatalf("error parsing INPUT_DRY-RUN: %s", err)
+		}
+	}
+
+	workingDir := flag.String("working-dir", os.Getenv("INPUT_WORKING-DIR"), "The path to the working directory.")
```

Comment:
Nit since this only happens if this is used outside of the action, but if the `INPUT_WORKING-DIR` env var isn't provided and `workingDir` gets set to empty string this doesn't end up being a reasonable default. What do you think about sticking with "./" being the default (same comment w/ `pull/main.go`)?

Reply:
Good point, fixed in b837294